### DISCUSSION TITLE
Cross-platform support

### DISCUSSION THREAD

**Discussion by lainq** [IndianOpenSourceFoundation/dynamic-cli#97] [Q&A]
Dynamic CLI is at present not supported for Windows. This problem is caused due to a package(`termios`) that is not implemented for Windows. Please help the organization find the right package 😐 .

Any sort of contribution is appreciated 🙂 !!

**Comment by Souvikns** [comment]
I found this https://github.com/veeso/termiWin

**Reply by GouravSardana** [reply]
This was written in C and it's not a pip package. I think we can't use it in our python project
